lbrynet/core/utils.py

```diff
@@ -9,6 +9,8 @@
 
 import pkg_resources
 
+from twisted.internet import reactor
```

Comment:
I have been curious about this - I've definitely seen example code where `reactor` was imported inside a function and I've never dug into why.  Guess this would be a good time to learn.